src/core/pyproxy.ts

```diff
@@ -424,12 +425,22 @@ export class PyProxyClass {
      * converts it to an array of entries, and ``(it) => new Map(it)`` converts
      * it to a ``Map`` (which is the default behavior).
      */
-    dict_converter?: any;
+    dict_converter?: (array: Iterable<[key: string, value: any]>) => any;
+    /**
+     * Optional argument to convert objects with no default conversion. See the
+     * documentation of :any:`pyodide.to_js`.
+     */
+    default_converter?: (
+      obj: PyProxy,
+      convert: (obj: PyProxy) => any,
+      cacheConversion: (obj: PyProxy, result: any) => void
+    ) => any;
```

Comment:
It looks like the [docstrings are not properly parsed](https://pyodide--2208.org.readthedocs.build/en/2208/usage/api/js-api.html#PyProxy.toJs). If this is an issue about compatibility between typescript and sphinx-js, we could fix this in a separate PR as there are some other functions where their docstrings are not properly parsed as well.

Reply:
I monkey patched in the handling for this so I probably made a mistake. It seems like sphinx-js is not Python 3.10 compatible (from ci in #2225) so we probably need to fork it soon.

Reply:
Thanks for noticing this.

Reply:
> so we probably need to fork it soon

Well, that's sad news 😥

Reply:
Actually, turns out I can monkey patch Python3.10 compatibility =)